lib/commands/getRepositoryUrl.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function (git) {
```

Comment:
`commands` folder should only contain commands

Reply:
should be done now